src/screens/discover/components/NetworkSwitcher.tsx

```diff
@@ -746,28 +755,24 @@ function Sheet({ children, header, onClose }: PropsWithChildren<{ header: ReactE
   );
 }
 
-export function NetworkSelector({
-  onClose,
-  multiple,
-}: {
-  onClose: (selected: ChainId[] | 'all') => void;
-  onSelect: VoidFunction;
-  multiple?: boolean;
-}) {
+export function NetworkSelector({ onClose }: { onClose: VoidFunction }) {
   const editing = useSharedValue(false);
-  const selected = useSharedValue<ChainId[] | 'all'>([]);
 
-  const close = () => {
-    'worklet';
-    runOnJS(onClose)(selected.value);
-  };
+  const network = useTrendingTokensStore(state => state.network);
+  const selected = useSharedValue<ChainId | undefined>(network);
+  const setNetwork = useTrendingTokensStore(state => state.setNetwork);
```

Comment:
I'm pretty sure this component is gonna be used in other places later, fine for now but lets try not to couple it too much

Reply:
decoupled it, thanks for the notice on this